runtime/Cargo.toml

```diff
@@ -249,6 +251,7 @@ std = [
     "pallet-offences/std",
     "pallet-preimage/std",
     "pallet-randomness-collective-flip/std",
+    "pallet-recovery/std",
```

Comment:
Probably we need to add `"pallet-recovery/try-runtime"` below if it has this feature

Reply:
I think it works without it

Reply:
Of course it works, but it is necessary for cases when we use `try-runtime`